### ISSUE TITLE
Limit nodes that ropebox can attach to

### ISSUE THREAD

**Issue by tacotexmex** [minetest-mods/ropes#4]
Because ropeboxes can be attached to themselves an infinite rope can be built. 

![skarmavbild 2017-04-30 kl 10 59 59](https://cloud.githubusercontent.com/assets/25328789/25563086/6c503022-2d94-11e7-8a74-12741e88ffc9.png)

**Comment by FaceDeer** [comment]
I've added a check to prevent rope boxes and rope ladders from being placed attached to nodes with walkable=false, I think that should cover most reasonable situations, how's that work for you?

Also, I've noticed that for some reason the textures of the rope ladder nodes are upside-down in the recent dev build of Minetest I use. But not in the 0.4.15 released version. Peculiar, I'll need to have a look at that when 0.4.16 comes out to see if I need to flip them.
